### PR TITLE
feat: add PR template

### LINKED ISSUE #1003: Set up a Pull Request template
As title says.

### PR DESCRIPTION
**Pull request by RobPasMue**
As title says. Closes #1003

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+## Issue linked
+**Please mention the issue number or describe the problem this pull request addresses.**
+
+## Checklist
+- [ ] I have tested my changes locally.
+- [ ] I have added necessary documentation or updated existing documentation.
+- [ ] I have followed the coding style guidelines of this project.
+- [ ] I have added appropriate unit tests.
+- [ ] I have reviewed my changes before submitting this pull request.
```
Comment (AlejandroFernandezLuces) [inline .github/pull_request_template.md:14]:
```suggestion
- [ ] I have reviewed my changes before submitting this pull request.
- [ ] I have linked the issue or issues that are solved to the PR if any.
- [ ] I have assigned this PR to myself.
```

**Review by AlejandroFernandezLuces — APPROVED** [review]
Love the idea! Minor suggestion

File: .github/pull_request_template.md (hunk 1 above)
Comment (RobPasMue) [reply .github/pull_request_template.md:14]: Yep, good additions! Thanks @AlejandroFernandezLuces

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- [ ] I have linked the issue or issues that are solved to the PR if any.
+- [ ] I have assigned this PR to myself.
+- [ ] I have made sure that the title of my PR follows (Conventional commits style)[https://www.conventionalcommits.org/en/v1.0.0/#summary] (e.g. ``feat: extrude circle to cylinder``)
```
Comment (RobPasMue) [inline .github/pull_request_template.md:15]:
```suggestion
- [ ] I have made sure that the title of my PR follows [Conventional commits style](https://www.conventionalcommits.org/en/v1.0.0/#summary) (e.g. ``feat: extrude circle to cylinder``)
```

**Review by AlejandroFernandezLuces — APPROVED** [review]

**Pull request merged**